### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#305] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152321920-0d910c06-3e22-4e83-8f75-dfb9b4b3af06.png)

**Comment by Exyss** [comment]
-- Risposta cancellata per i motivi indicati da Elia nel commento sotto --

**Comment by Elia-Belli** [comment]
La dimostrazione di @Exyss è algebrica e non fa utilizzo del Principio di Induzione, infatti nel passo induttivo è assente l'uso dell'ipotesi induttiva: procedendo in quel modo tanto vale operare sulla proposizione (equazione) iniziale.

**Dimostrazione per Induzione**
- **Caso Base**: $n=1$,  $2^2=4$;
- **Passo Induttivo**: $n\geq1$, $P(n) \Rightarrow P(n+1)$
   $P(n+1) = (n+2)^2-n^2 = 4(n+1) \rightarrow (n+2)^2-n^2 = 4n+4$ , ora applichiamo l'ipotesi induttiva al termine 4n
   $(n+2)^2-n^2 = (n+1)^2-(n-1)^2 + 4$ , adesso è possibile procedere algebricamente ottenendo: $4+4n = 4+4n$

**Comment by VladGeek** [comment]
La dimostrazione di @Elia-Belli è stata ritenuta sbagliata dal professor Piperno e quindi giudicata non valida a tutti quelli che l'abbiano svolta in quel modo nell'ultimo appello. La motivazione fornita dal professore è che non c'è l'induzione.
Allego il metodo che invece il professore ha ritenuto corretto.
![soluzione](https://github.com/sapienzastudentsnetwork/mmi2122/assets/58610173/2e113c2b-007e-4121-94e6-591121499b29)

**Comment by Exyss** [comment]
> La dimostrazione di @Elia-Belli è stata ritenuta sbagliata dal professor Piperno e quindi giudicata non valida a tutti quelli che l'abbiano svolta in quel modo nell'ultimo appello. La motivazione fornita dal professore è che non c'è l'induzione. Allego il metodo che invece il professore ha ritenuto corretto. ![soluzione](https://private-user-images.githubusercontent.com/58610173/309274128-2e113c2b-007e-4121-94e6-591121499b29.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkyMzc2MTgsIm5iZiI6MTcxOTIzNzMxOCwicGF0aCI6Ii81ODYxMDE3My8zMDkyNzQxMjgtMmUxMTNjMmItMDA3ZS00MTIxLTk0ZTYtNTkxMTIxNDk5YjI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI0VDEzNTUxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThmYTc1OTFhMTEwMzVmMGQyZGFjZTdkODc5ZDM1YTRiZGFmZjNjMjc1NjQ5ODAxZWZiMWVlOTljZWEzYzAyZDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oFlTU1uJBQKTXMzNLRb97MB_A2cvi2enff-aPU7pIEY)

Il metodo di @Elia-Belli è corretto, l'unico "errore" è nel non aver specificato quale sia l'ipotesi induttiva, cosa che in questo caso non è importante trattandosi di un'equivalenza.

In particolare, il metodo di Elia dimostra che $4n$ sia uguale a $(n+1)^2 - (n-1)^2$, mentre il professore "voleva" il contrario.

Tuttavia, entrambe le soluzioni ed entrambe le ipotesi sono equivalenti in quanto non c'è una precedenza all'interno di un'egualianza. Dunque, applicare l'ipotesi induttiva su $4n$ o su $(n+1)^2 - (n-1)^2$ non fa alcuna differenza.

Ipotizzare e affermare che $A = B$ è equivalente ad ipotizzare ed affermare che $B = A$.
